src/core/SetHash.pm6

```diff
@@ -203,6 +203,14 @@ my class SetHash does Setty {
     multi method Mixy (SetHash:D:) { self.MixHash }
 
 #--- interface methods
+    multi method STORE(SetHash:D: Setty:D \set --> SetHash:D) {
+        nqp::say("SetHash STORE from Setty") if nqp::getenvhash<RAKUDO_DEBUG>;
+        self.SET-SELF(
```

Comment:
Also, this new candidate doesn't need the `.is-lazy` check that the existing candidate has?

Reply:
I think a `Setty` cannot be lazy.